features/achordion.c

```diff
@@ -232,7 +258,7 @@ void achordion_task(void) {
   }
 
 #ifdef ACHORDION_STREAK
-  if (streak_timer && timer_expired(timer_read(), streak_timer)) {
+  if (streak_timer && timer_expired(timer_read(), (streak_timer + 800))) {
```

Comment:
Add a named constant for that `800`. Maybe

```c
#define MAX_STREAK_TIMEOUT 800
```

Reply:
I wasn't sure what the convention was on where to define this. I did it directly above.